plugin/src/main/java/jenkins/plugins/openstack/compute/UserDataVariableResolver.java

```diff
@@ -77,6 +78,16 @@
                 "Labels of the node.",
                 r -> r.labelString
         );
+        stub(
+                "SLAVE_KAFKA_SECRET",
```

Comment:
Please refrain from using the word `slave` and prefer `agent`. https://www.jenkins.io/blog/2020/06/18/terminology-update/

Reply:
I just want to be compliant with SLAVE_JNLP_SECRET variable.

Reply:
Yeah, we would have to rename that one as well.